src/sandbox.c

```diff
@@ -14,6 +14,14 @@ void sandbox_set_level(sandbox_level_t level) {
 	(void)level;
 }
 
+bool sandbox_enabled(void) {
+	return false;
+}
+
+bool sandbox_active(void) {
+	return false;
+}
+
 bool sandbox_enter(void) {
 	return true;
```

Comment:
I don't think so. The result is only used to determine if initialization failed and we should terminate the process. It would be more obvious with a proper type system. I can at least add a comment.